info/v1/container.go

```diff
@@ -443,6 +445,245 @@ type TcpStat struct {
 	Closing uint64
 }
 
+type TcpAdvanceStat struct {
```

Comment:
This is a large enough set of metrics that I think it deserves its own metrics type so that it can be enabled/disabled on its own.

Reply:
Fine, I will fix it